topics/icrc_28_trusted_origins.md

```diff
@@ -6,107 +6,63 @@
 <!-- TOC -->
 * [ICRC-28: Trusted Origins](#icrc-28-trusted-origins)
   * [Summary](#summary)
-  * [Identify](#identify)
-  * [Verify](#verify)
-    * [icrc28_get_trusted_origins](#icrc28_get_trusted_origins)
-  * [Supported standards](#supported-standards)
-    * [icrc28_supported_standards](#icrc28_supported_standards)
+  * [Methods](#methods)
+    * [icrc28_trusted_origins](#icrc28_trusted_origins)
   * [Use-Cases](#use-cases)
-    * [Hot Signer Use-Case](#hot-signer-use-case)
+    * [Wallet Delegation Use-Case](#wallet-delegation-use-case)
     * [Cold Signer Use-Case](#cold-signer-use-case)
 <!-- TOC -->
 
 ## Summary
 
-When a relying party wants to authenticate as a user, it uses
-a [session key](https://internetcomputer.org/docs/current/references/ic-interface-spec/#signatures), and by way of the
-authentication flow ([ICRC-34](./icrc_34_get_global_delegation.md) or [ICRC-57](./icrc_57_get_session_delegation.md))
-obtains a delegation chain that allows the session key to sign for the user's global or session identity.
+This standard describes how a canister can indicate that a relying party is trusted.
```

Comment:
The term relying party is not defined in this context. Also, the semantics of trusted are really unclear.

The longer I think about this, the more I'm convinced that we should make the endpoint use-case specific (i.e. session isolation) rather than a generic "trusted" one.

Reply:
I'm not sure we want to be prescriptive on use cases. Why not let the community explore what other use cases they could build with this?

Here's a proposed edit:
This standard describes how a canister can indicate that a relying party (an entity that relies on the canister for certain functions or services) is trusted.

A trusted relying party carries certain privileges, like for example the ability to request Account Delegations as per ICRC-34.

This standard does not describe how a canister should:
- Ensure that all entries are free from malicious intent.
- Handle entries that become malicious over time.